### PR TITLE
Gbe 625: Add conference picker to act tech info review list

### PR DESCRIPTION
**Pull request by jonkiparsky**

### CODE REVIEW

**Comment by jonkiparsky** [comment]
Closes #625

File: expo/gbe/report_views.py [hunk 1]
```diff
@@ -173,28 +175,36 @@ def personal_schedule(request, profile_id='All'):
...
+            pass
 
     return render(request,
                   'gbe/report/act_tech_review.tmpl',
                   {'this_show': show,
                    'acts': acts,
-                   'all_shows': conf.Show.objects.all()})
+                   'all_shows': conf.Show.objects.filter(
+                       conference=conference),
+                   'conference_slugs': conference_slugs()},
```
Comment (bethlakshmi) [inline expo/gbe/report_views.py]: When the user does the following:
1 - go to http://localhost:8282/reports
2 - select "Act Tech Info"
3 - pick an old conference

Then the conference picker defaults back to the current confernce.  If you send in the conference item, the picker will default show the currently selected conference.

**Comment by bethlakshmi** [comment]
Not sure where to point to in the code, but:

When the user does the following:
1 - go to http://localhost:8282/reports
2 - select "Act Tech Info"
3 - pick an old conference
4 - pick a show from the old conference
5 - the show is shown 
problem = the conference list defaults back to an upcoming conference, and not the conference the user is looking at.

The context of what the user is doing should stay the same.  Given the similarity of shows from year to year in GBE, it would be easy to loose track.

**Comment by bethlakshmi** [comment]
A last good thing would be that the conference picker on 
    http://localhost:8282/reports
has it's choice persisted to 
   http://localhost:8282/reports/acttechinfo/view_summary/

So that where ever the user has selected a conference, it stays consistent.

**Comment by bethlakshmi** [comment]
That's all I have.  I don't mind making a ticket for later on the last comment (consistency between reports and act tech summary).  But the part where the show is chosen and then selector flips back to a default conference will make using this really frustrating.

**Comment by jonkiparsky** [comment]
@bethlakshmi Done.

**Comment by jonkiparsky** [comment]
@bethlakshmi this should do what you want.

**Comment by bethlakshmi** [comment]
Sorry this took so long!!

**Pull request merged**